### PR TITLE
Added 'content-available' to the aps dictionary

### PR DESCRIPTION
**Pull request by Illizian**
I hope this meets with your approval.

I'm pretty sure I've followed PSR-2 and I have created basic test cases.

Thanks

### CODE REVIEW

File: tests/ZendService/Apple/Apns/MessageTest.php [hunk 1]
```diff
@@ -115,6 +115,19 @@ public function testSetSoundThrowsExceptionOnNonString()
...
 
+    public function testSetContentAvailableThrowsExceptionOnNonInteger()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        $this->message->setContentAvailable("string");
+    }
+
+    public function testGetContentAvailableReturnsCorrectInteger()
+    {
+        $value = 1;
```
Comment (Ocramius) [inline tests/ZendService/Apple/Apns/MessageTest.php]: Are values `!= 1` supposed to be allowed by the spec?

Comment (Illizian) [inline tests/ZendService/Apple/Apns/MessageTest.php]: @Ocramius are you referring to PSR2?

Comment (Ocramius) [inline tests/ZendService/Apple/Apns/MessageTest.php]: No, just about the apple push notifications allowed values for `content-available`

Comment (Illizian) [inline tests/ZendService/Apple/Apns/MessageTest.php]: Yeah, the only allowed value is 'number' (integer)

```
content-available | number | Provide this key with a value of 1 to indicate that new content is available. Including this key and value means that when your app is launched in the background or resumed, application:didReceiveRemoteNotification:fetchCompletionHandler: is called.
(Newsstand apps are guaranteed to be able to receive at least one push with this key per 24-hour window.)
```

Comment (Ocramius) [inline tests/ZendService/Apple/Apns/MessageTest.php]: Fair enuff, merging :-)

File: library/ZendService/Apple/Apns/Message.php [hunk 2]
```diff
@@ -283,6 +315,9 @@ public function getPayload()
         if (!is_null($this->sound)) {
             $aps['sound'] = $this->sound;
         }
+        if (!is_null($this->contentAvailable)) {
+            $aps['content-available'] = $this->contentAvailable;
```
Comment (Ocramius) [inline library/ZendService/Apple/Apns/Message.php]: This code-path is not actually merged

Comment (Illizian) [inline library/ZendService/Apple/Apns/Message.php]: Just added them in another commit faf7de5

**Comment by Ocramius** [comment]
@Illizian thanks!

**Comment by Illizian** [comment]
Thanks right-back-atcha!

Now to make my way up the extension chain :)

**Comment by Illizian** [comment]
@Ocramius Is it possible to push this up to [packagist](https://packagist.org/packages/zendframework/zendservice-apple-apns), as 1.0.4 is currently referencing 0ede952 and not 9a1a1b3

**Comment by Ocramius** [comment]
@Illizian please ping @mwillbanks about it

**Comment by mwillbanks** [comment]
@Ocramius @Illizian I'll see if we can package up a release here shortly, just need to check the issues list first :tea:

**Comment by Illizian** [comment]
@mwillbanks that'd be great! Any idea on time scales? I need this for a prototype that needs to be built for Friday.

**Pull request merged**